### ISSUE TITLE
Incredibly slow on my mac

### ISSUE THREAD

**Issue by ir-fuel** [betaflight/betaflight-configurator#1184]
I am running betaflight configurator 10.4.0, connected to an Eachine Wizard X220 BTFL 3.4.1 and it's incredibly slow. Everything has lag.

If for instance I go to the motors tab and spin up a motor, it takes more than 10 seconds before the motor actually spins. When I stop spinning the motor it takes another 10 seconds for that command to arrive to the quad.

I am using macOS 10.13.6 on a 2016 15 inch Macbook Pro Retina

**Comment by mikeller** [comment]
@ir-fuel: Have you tried restarting the configurator app?

**Comment by ir-fuel** [comment]
Yes. This is not a one time event. On my Windows 10 laptop it works fine
